WooCommerce/Classes/Model/MarkOrderAsReadUseCase.swift

```diff
@@ -18,15 +18,16 @@ struct MarkOrderAsReadUseCase {
         case unavailableNote
         case noNeedToMarkAsRead
     }
+
+#if canImport(Yosemite)
     /// Async method that marks the order note as read if it is the notification for the last order.
     /// We do it in a way that first we syncronize notification to get the remote `Note`
     /// and then we compare local `orderID` with the one from remote `Note`.
     /// If they match we mark it as read.
     /// Returns syncronized note if marking was successful and error if some error happened
-#if canImport(Yosemite)
     static func markOrderNoteAsReadIfNeeded(stores: StoresManager, noteID: Int64, orderID: Int) async -> Result<Note, Error> {
         let syncronizedNoteResult: Result<Note, Error> = await withCheckedContinuation { continuation in
-            let action = Yosemite.NotificationAction.synchronizeNotification(noteID: noteID) { syncronizedNote, error in
+            let action = NotificationAction.synchronizeNotification(noteID: noteID) { syncronizedNote, error in
```

Comment:
I thought maybe there was something odd with the namespace access that resulted in the build failure so I tried removing it. But that was not the reason.

If you find the code clear with the prefix, feel free to restore it.